### PR TITLE
use tidb-docker-compose project

### PR DESCRIPTION
**Pull request by tennix**
This PR add document to uses [tidb-docker-compose](https://github.com/pingcap/tidb-docker-compose) project for docker-compose deployment.
PTAL @sunhao2017 @lilin90 @iamxy

### CODE REVIEW

File: op-guide/docker-compose.md [hunk 1]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+
+3. 访问集群
+
+    ```bash
+    mysql -h 127.0.0.1 -P 4000 -u root
+    ```
+
+    访问集群 Grafana 监控页面： <http://localhost:3000> 默认用户名和密码都是 admin
+
+    [集群可视化页面](https://github.com/tidb-vision)： <http://localhost:8010>
```
Comment (iamxy) [inline op-guide/docker-compose.md]: 集群数据可视化

File: op-guide/docker-compose.md [hunk 2]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+
+    如果是 Mac 系统，也可以通过 homebrew 安装： `brew install kubernetes-helm`
+
+2. 下载 tidb-docker-compose
+
+    ```bash
+    git clone https://github.com/pingcap/tidb-docker-compose.git
+    ```
+
+3. 自定义配置
```
Comment (iamxy) [inline op-guide/docker-compose.md]: 自定义集群

File: op-guide/docker-compose.md [hunk 3]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
-```
-
-+ 使用 `docker-compose up -d` 创建并启动集群
-+ 使用 `docker-compose port tidb 4000` 获得 TiDB 的对外服务端口。由于 Docker 做了端口转发, 你可能会获得一个类似 `0.0.0.0:32966` 的结果, 那么就可以通过 `32966` 端口访问 TiDB
-+ 运行 `mysql -h 127.0.0.1 -P 32966 -u root -D test` 连接到 TiDB 进行测试
-+ 运行 `docker-compose down` 关闭并销毁集群
\ No newline at end of file
+# 使用 Docker Compose 快速构建集群
+
+[Docker compose](https://docs.docker.com/compose/overview) 可以通过一个 YAML 文件定义多个容器的应用服务，然后一键启动或停止。可以用来在单机上一键部署一套 TiDB 测试集群，使用 Docker Compose 部署 TiDB 集群要求 docker 是 17.06.0 及以上版本
```
Comment (lilin90) [inline op-guide/docker-compose.md]: 1. Docker compose -> Docker Compose (Please keep the expression consistent)
2. docker -> Docker
3. Add a period at the end of the last sentence.

File: op-guide/docker-compose.md [hunk 4]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+    cd tidb-docker-compose && docker-compose up -d
+    ```
+
+3. 访问集群
+
+    ```bash
+    mysql -h 127.0.0.1 -P 4000 -u root
+    ```
+
+    访问集群 Grafana 监控页面： <http://localhost:3000> 默认用户名和密码都是 admin
```
Comment (lilin90) [inline op-guide/docker-compose.md]: Delete the extra whitespace after the colon.

File: op-guide/docker-compose.md [hunk 5]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+
+    [集群数据可视化](https://github.com/tidb-vision)： <http://localhost:8010>
```
Comment (lilin90) [inline op-guide/docker-compose.md]: Ditto

File: op-guide/docker-compose.md [hunk 6]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+
+## 自定义集群
+
+快速入门里面默认部署 3 个 PD，3 个 TiKV，1 个 TiDB 和监控组件 prometheus, pushgateway, grafana 以及 tidb-vision。如果想自定义集群，可以直接修改 docker-compose.yml，但是手动修改比较繁琐而且容易出错，强烈建议使用 [Helm](https://helm.sh) 模板引擎生成 docker-compose.yml 文件。
```
Comment (lilin90) [inline op-guide/docker-compose.md]: 1. “快速入门” -> “快速开始”？
2. "prometheus, pushgateway, grafana" -> "Prometheus，Pushgateway，Grafana"

Pls use the full-width punctuation in Chinese documents.

File: op-guide/docker-compose.md [hunk 7]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+
+1. 安装 Helm
+
+    [Helm](https://helm.sh) 可以用作模板渲染引擎，只需要下载其 binary 文件即可以使用
+
+    ```bash
+    curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
+    ```
+
+    如果是 Mac 系统，也可以通过 homebrew 安装： `brew install kubernetes-helm`
```
Comment (lilin90) [inline op-guide/docker-compose.md]: homebrew -> Homebrew

File: op-guide/docker-compose.md [hunk 8]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+    docker-compose -f generated-docker-compose.yml up -d
+    ```
+
+6. 访问集群
+
+    ```bash
+    mysql -h 127.0.0.1 -P 4000 -u root
+    ```
+
+    访问集群 Grafana 监控页面： <http://localhost:3000> 默认用户名和密码都是 admin
```
Comment (lilin90) [inline op-guide/docker-compose.md]: Delete the extra whitespace after the colon.

File: op-guide/docker-compose.md [hunk 9]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+    cd tidb-docker-compose
+    cp compose/values.yaml values.yaml
+    vim values.yaml
+    ```
+
+    修改 values.yaml 里面的配置，例如集群规模，TiDB 镜像版本等
+
+    [tidb-vision](https://github.com/pingcap/tidb-vision) 是 TiDB 集群可视化页面，可以可视化地显示 PD 对 TiKV 数据的调度。如果不想部署该组件，可以将 `tidbVision` 项留空。
+
+    pd, tikv, tidb 和 tidb-vision 支持从 GitHub 源码或本地文件构建 docker 镜像，供开发测试使用。
```
Comment (lilin90) [inline op-guide/docker-compose.md]: 1. PD，TiKV，TiDB
    Pls update the other similar items involved in this PR, and note the punctuation.
2. -> Docker

File: op-guide/docker-compose.md [hunk 10]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
+
+6. 访问集群
+
+    ```bash
+    mysql -h 127.0.0.1 -P 4000 -u root
+    ```
+
+    访问集群 Grafana 监控页面： <http://localhost:3000> 默认用户名和密码都是 admin
+
+    如果启用了 tidb-vision，可以通过 <http://localhost:8010> 查看
```
Comment (lilin90) [inline op-guide/docker-compose.md]: Pls add a period at the end.

File: op-guide/docker-compose.md [hunk 11]
```diff
@@ -3,156 +3,92 @@ title: 使用 Docker Compose 构建集群
...
-
-    privileged: true
-```
-
-+ 使用 `docker-compose up -d` 创建并启动集群
-+ 使用 `docker-compose port tidb 4000` 获得 TiDB 的对外服务端口。由于 Docker 做了端口转发, 你可能会获得一个类似 `0.0.0.0:32966` 的结果, 那么就可以通过 `32966` 端口访问 TiDB
-+ 运行 `mysql -h 127.0.0.1 -P 32966 -u root -D test` 连接到 TiDB 进行测试
-+ 运行 `docker-compose down` 关闭并销毁集群
\ No newline at end of file
+# 使用 Docker Compose 快速构建集群
```
Comment (lilin90) [inline op-guide/docker-compose.md]: Pls add this file to README.md

**Comment by iamxy** [comment]
LGTM

**Pull request merged**